tests/unit/test_slack.py

```diff
@@ -166,14 +166,18 @@ def test_notify_automatic_processing_label(mocker, monkeypatch):
 
     notifier.notify_automatic_processing(
         ProductInsight(
-            barcode="123", source_image="/image/1", type="label", value_tag="en:vegan"
+            barcode="123",
+            source_image="/image/1",
+            type="label",
+            value_tag="en:vegan",
+            data={"bounding_box": (2, 2, 4, 4)},
```

Comment:
There again you do not have to change this test.

There is no bounding_box in this case.